contact-us/contact-list.html

```diff
@@ -22,13 +22,23 @@
                 <button class="js-type-and-filter_button btn btn__super">Filter</button>
             </div>
         </div>
-        
+
         <div class="helpful-terms">
             <span class="helpful-terms_header">Helpful filter terms:</span>
-            <button class="btn btn__link js-helpful-term">Regulations</button>
-            <button class="btn btn__link js-helpful-term">Compliance guides</button>
-            <button class="btn btn__link js-helpful-term">Mortgages</button>
-            <button class="btn btn__link js-helpful-term">Consumer help</button>
+            <ul class="list list__unstyled helpful-terms_list">
```

Comment:
Possibly you could remove the repetition in the list by using a jinja loop to generate the list items and buttons:

```
<ul class="list list__unstyled helpful-terms_list">
    {% for term in ['Regulations', 'Compliance guides', 'Mortgages', 'Consumer help'] %}
    <li class="list_item">
        <button class="btn btn__link js-helpful-term">{{term}}</button>
    </li>
    {% endfor %}
</ul>
```

The actual array of terms could actually be pulled out into a variable in `_vars-contact-us.html` perhaps.